README.md

```diff
@@ -119,6 +119,10 @@ Note that the 2 Egeria React UI npm packages are not, at this time, published to
 
 The version in the package.json can be updated using he command 'npm version preminor -preid rc' or can be editted directly in the package.json file.
 
+## SSL configuration
+
+By default the Egeria React UI uses a keystore.p12 file in ssl, this is a copy of file 'https://github.com/odpi/egeria/blob/master/keystore.p12'. These files should be kept the same. This keystore file allows Egeria to run securely in a demo environment; it is not appropriate for production, which should be appropriately secured.      
+
```

Comment:
Looking at the code it does seem as if trust store is indeed appropriate as this is the store needed to check for known certificates/certificate authorities when the server you are connecting to presents them. It's probably more commonly used in this context afaik. (we may have it wrong in places in egeria)

Now I don't see where the config is to decide whether the client code here should present a certificate to the server for validation , which would be used for 2 way SSL. 

Also in this UI what certificate is presented back to the browser user? (that would normally be in the keystore). it's fine to be the same one by default, but it should be configurable to be different one. Typically you'd also expect configuration to decide which certificate to use since a keystore may hold many. Plus the password to access the keystore.



The only challenge may be technically for 2 way ssl you'd also need a truststore to validate the

Reply:
> 'kept the same' -> 'kept the same if you are using the provided self-signed certificate'.
> 
> 'securely in a demo' -> 'simply in a demo/development'
> 
> '; it is not ..... secured' -> For production use refer to https://egeria-project.org/open-metadata-implementation/admin-services/docs/user/omag-server-platform-transport-level-security.html which provides more information'
> 
> Side note - I'm not clear yet (reviewing code) of your use of this file.
> 
> * keystore stores certificates that an app uses to establish a connection (ie as a client to a backend) or to respond (2 way SSL)
> * truststore stores certificates, or more typically certificate authorities, that are used to validate certificates received from the other party
> 
> Ideally you need both configurable.

Fixed

Reply:
@planetf1 I agree the truststore looks best for server authentication. I will change that for now. It would be good to have both.